www/js/views/item.js

```diff
@@ -59,7 +59,7 @@ ItemView = Backbone.View.extend({
         buttons = this.getButtons() === false ? false: buttons;
 
         var item = _.template($(template).html())( {
-            name: this.model.get('formName'),
+            name: this.model.escape('formName'),
```

Comment:
Is `model.escape` defined in Backbone?

Reply:
@pb82 yes, http://backbonejs.org/#Model-escape. From the documentation:

"Similar to get, but returns the HTML-escaped version of a model's attribute. If you're interpolating data from the model into HTML, using escape to retrieve attributes will prevent XSS attacks."